vita3k/io/src/io.cpp

```diff
@@ -71,6 +71,15 @@ bool read_app_file(FileBuffer &buf, const std::string &pref_path, const std::str
     return read_file(VitaIoDevice::ux0, buf, pref_path, fs::path("app") / title_id / vfs_file_path);
 }
 
+SpaceInfo get_space_info(const VitaIoDevice device, const std::string &vfs_path, const std::string &pref_path) {
+    SpaceInfo space_info;
+    const auto host_path = device::construct_emulated_path(device, vfs_path, pref_path);
+    space_info.max_capacity = fs::space(host_path).capacity;
+    space_info.used = fs::space(host_path).capacity - fs::space(host_path).available;
```

Comment:
Fill free first, then subtract max capacity with free that already called to get. I think its just more efficient, the effect is probablly not seenable though.